packages/replay/metadata/source.ts

```diff
@@ -71,6 +71,12 @@ function getCircleCIMergeId(env: NodeJS.ProcessEnv) {
   }
 }
 
+function getBuildkiteMessage(env: NodeJS.ProcessEnv) {
+  if (env.BUILDKITE_SOURCE === "webhook") {
```

Comment:
Just double checking, do we want to support any other sources? https://buildkite.com/docs/pipelines/environment-variables#BUILDKITE_SOURCE

Reply:
Starting with just `webhook` based on the conversation on Linear